test/libsolidity/syntaxTests/natspec/invalid/docstring_inherit_modifier_no_return.sol

```diff
@@ -0,0 +1,14 @@
+contract C {
+	///@return
+	modifier m22 {}
+}
+
+contract D is C {
+	modifier m22 {}
+}
+// ----
+// SyntaxError 2883: (39-41): Modifier body does not contain '_'.
+// SyntaxError 2883: (77-79): Modifier body does not contain '_'.
```

Comment:
Maybe add `_` in the modifiers to get rid of those errors, since they're unrelated?